cmd/telegraf/cmd_config.go

```diff
@@ -32,6 +33,60 @@ func getConfigCommands(pluginFilterFlags []cli.Flag, outputBuffer io.Writer) []*
 				return nil
 			},
 			Subcommands: []*cli.Command{
+				{
+					Name:  "check",
+					Usage: "check configuration file(s) for issues",
+					Description: `
+		The 'check' command reads the configuration files specified via '--config' or
+		'--config-directory' and tries to initialize, but not start, the plugins.
+		Syntax and semantic errors detectable without starting the plugins will
+		be reported.
+		If no configuration file is	explicitly specified the command reads the
+		default locations and uses those configuration files.
+
+		To check the file 'mysettings.conf' use
+
+		> telegraf --config mysettings.conf config check
```

Comment:
This looks like a backwards way of doing this. I would have expected:

> telegraf config check --config...

I know you are trying to re-use the global config/config-dir variables, but is there a way to have these so they are local to the check command?